deps/rabbitmq_prometheus/priv/schema/rabbitmq_prometheus.schema

```diff
@@ -141,8 +141,6 @@ end}.
 {mapping, "prometheus.ssl.max_keepalive", "rabbitmq_prometheus.ssl_config.cowboy_opts.max_keepalive",
     [{datatype, integer}, {validators, ["non_negative_integer"]}]}.
 
-{mapping, "prometheus.filter_aggregated_queue_metrics_pattern", "rabbitmq_prometheus.filter_aggregated_queue_metrics_pattern", [{datatype, string}]}.
```

Comment:
very true, thanks for spotting this. (a deprecated attribute as proposed here would be useful in this case https://github.com/Kyorai/cuttlefish/issues/39)

I wonder if this PR could be split into two: removing this mapping from the schema is 4.0 only and is not backported from main, but removing the implementation is a separate PR/commit and can be backported to 3.13.x. Otherwise it makes it a bit more difficult to backport any later changes to the core metrics module (ie. any new metrics or rather new metric family). Is this reasonable?